frontend/src/routes/LandingPage/Admission.tsx

```diff
@@ -0,0 +1,283 @@
+import React, { useEffect, useState } from "react";
+import styled from "styled-components";
+import djangoData from "src/utils/djangoData";
+import LegoButton from "src/components/LegoButton";
+import { media } from "src/styles/mediaQueries";
+import FormatTime from "src/components/Time/FormatTime";
+import { useMyApplication } from "src/query/hooks";
+import { Admission as AdmissionInterface } from "src/types";
+import CountDown from "./CountDown";
+
+interface AdmissionProps {
+  admission: AdmissionInterface;
+}
+
+const Admission: React.FC<AdmissionProps> = ({ admission }) => {
+  const [hasSubmitted, setHasSubmitted] = useState(false);
+
+  const { data: myApplication } = useMyApplication(admission.pk);
+
+  useEffect(() => {
+    setHasSubmitted(!!myApplication);
+  }, [myApplication]);
+
+  return (
+    <AdmissionWrapper>
```

Comment:
Hopefully right line; this is pain doing on the phone 🙃

Could the white "background sheet" have round corners, like the apply-button? 

Also, the "Søknadsfrist ..." should be a bit more distinguished from the other text under imo, maybe bold or something.

Reply:
Personally not that big a fan of having round corners on cards of that size. Did you want it to have the corners rounded as much as the button, or could less do?

Reply:
I think you can do less. It looks more like a card when it has rounded corners, which makes sense to me. But only add it if you like it :)